### DISCUSSION TITLE
PNG con fondo transparente si o no?

### DISCUSSION THREAD

**Discussion by PatricioRios** [ArisGuimera/JetpackComposePro#84] [General]
vi que en todos los composes en la imagen de el componente tiene un fondo, oscuro o claro (dependiendo de la persona),
seria mejor colocar imagenes con el compose, y un fondo transparente?, de esta manera se adaptaría mejor a el modo claro y oscuro, aunque seria mejor hacer una foto para cada uno de ellos, pero no se como se hace.

**Comment by ArisGuimera** [comment]
Se pueden tener imágenes que cambien al usar el modo claro u oscuro. Aunque a mi lo que más me gusta fue lo que hicieron en esta PR

https://www.jetpackcompose.pro/sheets/bottom-sheet/

Creo que quedó genial y va bastante bien. ¿Qué os parece hacer algo así?

**Comment by jhordyhuaman** [comment]
> Se pueden tener imágenes que cambien al usar el modo claro u oscuro. Aunque a mi lo que más me gusta fue lo que hicieron en esta PR
> 
> https://www.jetpackcompose.pro/sheets/bottom-sheet/
> 
> Creo que quedó genial y va bastante bien. ¿Qué os parece hacer algo así?

Hola Aristis, las imágenes correspondiente a esa PR los genere con esta herramienta  https://hotpot.ai/ los dejo la plantilla si por si lo quiere usar: https://hotpot.ai/design/google-play-feature-graphic?id=JgIPUDyJJlcn_400_700

**Reply by ArisGuimera** [reply]
Muchas gracias! Seguramente me vaya a tocar a mi al final unificar todas las imágenes y esta web nos puede venir genial!
